app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  def three_most_recent_records
+    posts.order(created_at: :desc).first(3)
+  end
+end
```

Comment:
- [ ] a user should be able to have many comments and likes as well similar to the posts.